src/components/NavMenu/NavMenu.jsx

```diff
@@ -1,91 +1,107 @@
 /* eslint-disable react/require-default-props */
 import React, { useState, createElement } from 'react';
-import { Link, useLocation } from 'react-router-dom';
+import { Link, useLocation, useNavigate } from 'react-router-dom';
 import { Menu, Layout, Button } from 'antd';
-import PropTypes from 'prop-types';
+import { instanceOf } from 'prop-types';
 import {
   ProfileOutlined,
   TableOutlined,
   DashboardOutlined,
   FormOutlined,
   MenuUnfoldOutlined,
   MenuFoldOutlined,
+  LogoutOutlined,
+  UserOutlined,
 } from '@ant-design/icons';
 import Logo from '../../assets/Logo.png';
 import styles from './NavMenu.module.css';
 import './NavAnt.css';
 
+import { Cookies, withCookies, cookieKeys } from '../../util/cookie_utils';
```

Comment:
pls move these util imports up so that the styles are imported last (for consistency)